participants/Rajpoot/pr_tutorial/tests/test_buggy_function.py

```diff
@@ -0,0 +1,6 @@
+from pr_tutorial.buggy_function import angle_to_sexagesimal
+
+def test_angle_to_sexigesimal():
+    """Simplest test for one case"""
+
+    assert angle_to_sexagesimal(66.8, decimals=3) == '66° 48′ 0″'
```

Comment:
This test will fail, since the function converts to hours and not degrees.

```suggestion
    assert angle_to_sexagesimal(66.8, decimals=3) == "04:27:12.000"
```